Ant 1.10.14 does not "get along" with EPP

Since Eclipse Platform update SimRel to a version containing Ant 1.10.14 the EPP has failed to build:

```java
19:33:57 Caused by: org.eclipse.aether.transfer.ArtifactNotFoundException: Could not find artifact org.eclipse.orbit:ant:jar:1.10.14 in eclipse.maven.central.mirror (https://repo.eclipse.org/content/repositories/maven_central/)
19:33:57     at org.eclipse.aether.connector.basic.ArtifactTransportListener.transferFailed (ArtifactTransportListener.java:48)
19:33:57     at org.eclipse.aether.connector.basic.BasicRepositoryConnector$TaskRunner.run (BasicRepositoryConnector.java:369)
19:33:57     at org.eclipse.aether.util.concurrency.RunnableErrorForwarder$1.run (RunnableErrorForwarder.java:75)
19:33:57     at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1136)
19:33:57     at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:635)
19:33:57     at java.lang.Thread.run (Thread.java:833)
```

--- a/releng/org.eclipse.epp.config/parent/pom.xml
+++ b/releng/org.eclipse.epp.config/parent/pom.xml
@@ -94,6 +94,14 @@
       <url>https://download.eclipse.org/justj/epp/milestone/latest/</url>
       <layout>p2</layout>
     </repository>
+    <repository>
+      <!-- Tycho is insisting on accessing the "upstream" artifact for this Ant, see https://github.com/eclipse-packaging/packages/issues/66
+           To make this work we need to match the repo used in Orbit's build for the repository:
+             https://github.com/eclipse-orbit/orbit-simrel/blob/f6ea491f4ada5d6d6750d8118bf40d9802993376/maven-bnd/tp/MavenBND.target#L1203
+      -->
+      <id>maven-ant-repository2</id>
+      <url>https://repo.eclipse.org/content/repositories/orbit-approved-artifacts</url>
+    </repository>
   </repositories>
 
   <build>

